scripts/README.md

```diff
@@ -291,3 +291,27 @@ function build_release() {
 
 main $@
 ```
+### Using release.sh to publish to Azure
+To run release.sh to publish to Azure Contianer Registery and Azure Blob, the following pre-requisites need to be configured/setup:
```

Comment:
nit: for readability sake, instead of subsections, these steps can be a simple list